feature-libs/asm/core/connectors/asm.adapter.ts

```diff
@@ -9,7 +9,7 @@ import { Observable } from 'rxjs';
 import {
   CustomerSearchOptions,
   CustomerSearchPage,
-} from '../models/asm.models';
+} from '../../root/model/asm.models';
```

Comment:
This import is invalid.  It should be `@spartacus/asm/root`.  There are many other instances of this.  I'd recommend running a search for "../root/model" in the asm folder.

This file (`asm.adapter.ts`) is part of the `@spartacus/asm/core` module.  Relative imports will only work for files that are part of the same module.  The `asm.models.ts` file was moved to the `@spartacus/asm/root` module, so we need to use a fully qualified import.  The auto-import is not smart enough to recognize this and gave you the same bad import for all the files.

This issue would only appear once you tried to compile the Spartacus project into the Angular library using `npm build:libs`.

Reply:
Ah, I see, thanks very much for providing more details, and now I have corrected it.

Reply:
> This is looking pretty good. Just the imports to fix and then documenting the API changes in the migration document.
the API changes have been added in the migration document.